doc/book/admin/upgrades/upgrade_cluster.rst

```diff
@@ -0,0 +1,323 @@
+
+..  _admin-upgrades-replication-cluster:
+
+Replication cluster upgrade
+===========================
+
+Below are the general instructions for upgrading a Tarantool cluster with replication.
+Upgrading from some versions can involve certain specifics. To find out if it is your
+your case, check the version-specific topics of the :ref:`Upgrades <admin-upgrades>`
+section.
+
+A replication cluster can be upgraded without downtime due to its redundancy.
+When you disconnect a single instance for an upgrade, there is always another
+instance that takes over its functionality: being a master storage for the same
+data buckets or work as a router. This way, you can upgrade all the instances one by one.
+
+The high-level steps of cluster upgrade are the following:
+
+#.  :ref:`Ensure the application compatibility <admin-upgrades-cluster-check-app>` with
+    the target Tarantool version.
+#.  :ref:`Check the cluster health <admin-upgrades-cluster-pre-check>`.
+#.  :ref:`Install the target Tarantool version <admin-upgrades-cluster-install>` on the
+    cluster nodes.
+#.  :ref:`Upgrade router <admin-upgrades-cluster-routers>` nodes one by one.
+#.  :ref:`Upgrade storage replica sets <admin-upgrades-cluster-storages>` one by one.
+
+
+..  important::
+
+    The only way to upgrade Tarantool from version 1.6, 1.7, or 1.9 to 2.x **without downtime** is
+    taking an intermediate step by upgrading to 1.10 and then to 2.x.
+
+    Before upgrading Tarantool from 1.6 to 2.x, please read about the associated
+    :ref:`caveats <admin-upgrades-1.6-1.10>`.
+
+.. note::
+
+    Some upgrade steps are moved to the separate section :ref:`Procedures and checks <upgrade_cluster-procedures>`
+    to avoid overloading the general instruction with details. Typically, these are
+    checks you should repeat during the upgrade to ensure it goes well.
+
+If you experience issues during upgrade, you can roll back to the original version.
```

Comment:
Just an idea - place this as the 6th step of the introduction

Reply:
I'd leave it separate so that the instruction steps form the success scenario.